packages/override-tools/src/FileRepositoryInterface.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+/**
+ * Provides access to patch files
+ */
+export interface IOverrideFileRepository {
```

Comment:
>I [](start = 17, length = 1)

nit: generally JavaScript and Java communities do not use 'I' as a prefix for interfaces. Even TypeScript team internally does not use it: [https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)

Reply:
I can change this. For reference, Fabric does actually use the `I` prefix in TS, but it does feel a bit weird.